lib/Mailer.php

```diff
@@ -246,10 +262,17 @@ private function doSend(MessageInterface $message)
 
             $content = $builder->build();
             $headers['Content-Type'] = 'multipart/form-data; boundary="'.$builder->getBoundary().'"';
+        } else {
+            $content = $this->getStreamFactory()->createStream($content);
```

Comment:
Lets do `if (empty($files))` then this logic. Ie, the smaller block should be first. That helps readability.